keyboards/beekeeb/piantor/rules.mk

```diff
@@ -1 +1,3 @@
 SERIAL_DRIVER = vendor
+
+LAYOUTS = split_3x6_3
```

Comment:
As the rest is in `info.json`, this change should also be made there too

```json
community_layouts": ["split_3x6_3"],
```

```suggestion
```

Reply:
Thank you for clarifying that. Updated.